### PR TITLE
readme updates

### PR DESCRIPTION
**Pull request by hervenicol**
This PR updates README.
Towards https://github.com/giantswarm/dashboards/pull/243

### Checklist

- [x] Update changelog in CHANGELOG.md in an end-user friendly language.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
 # Dashboards
 
-This project currently contains Giant Swarm public dashboards.
+This project currently contains Giantswarm public dashboards.
```
Comment (TheoBrigitte) [inline README.md:3]: Giant Swarm is actually correct

File: README.md [hunk 2]
```diff
@@ -46,3 +47,15 @@ To Update the `kubernetes-mixin` dashboards:
...
+
+Some dashboards are crafted by us (Giantswarm) or forked from public dashboards.
+
+Some dashboards come from mixins, with a few manual updates. They have a `.*-mixin` tag, like `kubernetes-mixin` or `prometheus-mixin`.
+
+Beware when updating them: even though they come from mixins, they may require tweaks to work with our architecture.
+
+The dashboards that don't require tweaks can be automatically updated from pre-generated mixins at https://monitoring.mixins.dev/ with the `update-mixins-monitoring` github workflow.
+
+In the future we should provide documentation and automation to perform clean updates of mixins-based dashboards.
```
Comment (TheoBrigitte) [inline README.md:61]: As you mentioned before we seemed to have automation for updating mixins. Or I don't get what we are talking about here?

File: README.md (hunk 1 above)
Comment (hervenicol) [reply README.md]: ok, fixed.

File: README.md (hunk 2 above)
Comment (hervenicol) [reply README.md:61]: Well... for some, we have scripts, that may work. Or not. Depends if people have manually updated the dashboards in the meantime.
So my idea of the procedure is:
- `git log` or `git blame` to check what/when the dashboards where changed.
- run the script
- look at diff
- cross fingers

But according to the readme, the script `./scripts/sync-mixins.sh` is for grafana cloud dashboards, which means you can't even `git blame`. I think there's a changelog though.

There's too much unknown there, I prefer not to dive in for the moment, that's why I didn't get into details.

Comment (TheoBrigitte) [reply README.md:61]: I like the crossing finger part 😁

Ok this is more like warning that the automation is not perfect.

**Review by TheoBrigitte — APPROVED** [review]
Fine with me

**Pull request merged**